wandb/data_types.py

```diff
@@ -1027,25 +1035,33 @@ class Html(BatchableMedia):
             to False the HTML will pass through unchanged.
     """
 
+    artifact_type = "html-file"
+    html_prefix = '<base target="_blank"><link rel="stylesheet" type="text/css" href="https://app.wandb.ai/normalize.css" />'
```

Comment:
Where is this used?

Reply:
I just removed it 😂

Reply:
I was refactoring the constructor, but realized it was not needed and forgot to clean it up